fedimint-rocksdb/src/lib.rs

```diff
@@ -35,6 +36,18 @@ fn is_power_of_two(num: usize) -> bool {
     num.count_ones() == 1
 }
 
+impl<'a> fmt::Debug for RocksDbReadOnlyTransaction<'a> {
```

Comment:
~`MemTransaction` currenlty derives `Debug`, resulting in a lot of detail being printed. Maybe we want to thin it out a bit if DBTX are going to be logged a lot.~

I see you did so in the next commit :laughing: